pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -431,12 +431,17 @@ type ObjectStoreSpec struct {
 	// The multisite info
 	Zone ZoneSpec `json:"zone"`
 
-	// The rgw endpoint healthcheck
+	// The rgw daemon healthchecks and liveness probe
 	HealthCheck BucketHealthCheckSpec `json:"healthCheck"`
 }
 
 type BucketHealthCheckSpec struct {
-	Bucket HealthCheckSpec `json:"bucket,omitempty"`
+	DaemonHealth  CephObjectDaemonHealthSpec           `json:"daemonHealth,omitempty"`
```

Comment:
Please revert, @travisn and I had a long debate on the naming, we better not resurrect it :-p

Reply:
okay. Done